docs_app/content/marketing/team.html

```diff
@@ -1 +1,2 @@
+<h1 class="no-toc">Team</h1>

```

Comment:
I think this heading is not needed. The reason for this is that the heading is still visible in the top header:

<img width="999" alt="image" src="https://github.com/ReactiveX/rxjs/assets/28087049/4b3f0b2d-daea-4e8a-a62e-8b0adc438246">

Please revert this one.